cmd/server-main.go

```diff
@@ -503,7 +487,7 @@ func serverMain(ctx *cli.Context) {
 	globalObjectAPI = newObject
 	globalObjLayerMutex.Unlock()
 
-	go startBackgroundOps(GlobalContext, newObject)
+	go initDataCrawler(GlobalContext, newObject)
```

Comment:
We can wait staring this until we are out of safe mode, so move down until after?